config/include.go

```diff
@@ -367,6 +367,8 @@ func (targetConfig *TerragruntConfig) DeepMerge(sourceConfig *TerragruntConfig,
 	for key, val := range sourceConfig.GenerateConfigs {
 		targetConfig.GenerateConfigs[key] = val
 	}
+
+	targetConfig.ProcessedIncludes = sourceConfig.ProcessedIncludes
```

Comment:
Also remove this line, for the same reason it was removed from the `Merge` function.